FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -103,57 +114,41 @@ TrajectorySeedProducer::TrajectorySeedProducer(const edm::ParameterSet& conf):
     }
 
     originRadius = conf.getParameter<double>("originRadius");
-
-    //collections to read in
-    std::vector<edm::InputTag> defaultSkip;
-    std::vector<edm::InputTag> skipSimTrackIdTags = conf.getUntrackedParameter<std::vector<edm::InputTag> >("skipSimTrackIdTags",defaultSkip);
-    for ( unsigned int k=0; k<skipSimTrackIdTags.size(); ++k ) { 
-       skipSimTrackIdTokens.push_back(consumes<std::vector<int> >(skipSimTrackIdTags[k]));
-    }
-
     originHalfLength = conf.getParameter<double>("originHalfLength");
-
     originpTMin = conf.getParameter<double>("originpTMin");
```

Comment:
@shchauha 

please change variable name 
originpTMin -> ptMin 
to follow same convention as full tracking